src/status_im2/contexts/syncing/how_to_pair/style.cljs

```diff
@@ -0,0 +1,101 @@
+(ns status-im2.contexts.syncing.how-to-pair.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def scroll-view
+  {:flex             1
+   :background-color colors/neutral-95})
+
+(def container-outer
+  {:style {:background-color colors/neutral-95
+           :justify-content  :space-between
+           :align-items      :left
+           :padding          16
+           :border-radius    10}})
+
+(def container-instructions
+  {:flex               1
+   :padding-top        20
+   :padding-bottom     8
+   :padding-horizontal 16
+   :background-color   colors/neutral-95})
+
+(def heading
+  {:size   :heading-1
+   :weight :semi-bold
+   :style  {:margin-bottom 12
```

Comment:
same here, remove style key and use in view file :)

Reply:
done